reward/reward_config_cache_test.go

```diff
@@ -92,6 +92,10 @@ func (governance *testGovernance) StakingUpdateInterval() uint64 {
 	return governance.stakingInterval
 }
 
+func (governance *testGovernance) GetMinimumStakingAtNumber(num uint64) (uint64, error) {
+	return 5000000, nil
```

Comment:
It is minor, but how about using `DefaultMinimumStake` instead of magic number?

Reply:
Fixed in c12e0861.